source/_integrations/microsoft.markdown

```diff
@@ -41,6 +41,11 @@ type:
   required: false
   type: string
   default: "`ZiraRUS`"
+output:
+  description: "Format of audio output. Accepted values are listed [in the documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/rest-text-to-speech#audio-outputs)."
+  required: false
+  type: string
+  default: "`audio-16khz-128kbitrate-mono-mp3`"
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>